firebase.json

```diff
@@ -16,43 +16,38 @@
     ],
     "redirects": [
       {
-        "source": "/tos.html",
-        "destination": "/terms",
-        "type": 301
-      },
-      {
-        "source": "/articles/benchmarking/",
-        "destination": "/articles/dart-vm/benchmarking",
+        "source": "/+",
+        "destination": "https://plus.sandbox.google.com/+dartlang",
         "type": 301
       },
       {
-        "source": "/articles/io/",
-        "destination": "/articles/dart-vm/io",
+        "source": "/+dart2js",
+        "destination": "https://webdev.dartlang.org/tools/dart2js",
         "type": 301
       },
       {
-        "source": "/articles/native-extensions-for-standalone-dart-vm/",
-        "destination": "/articles/dart-vm/native-extensions",
+        "source": "/+isolates",
+        "destination": "https://dart-lang.github.io/observatory/isolate.html",
         "type": 301
       },
       {
-        "source": "/articles/numeric-computation/",
-        "destination": "/articles/dart-vm/numeric-computation",
+        "source": "/+lexicalscope",
+        "destination": "/guides/language/language-tour#lexical-scope",
         "type": 301
       },
       {
-        "source": "/articles/simd/",
-        "destination": "/articles/dart-vm/simd",
+        "source": "/+pub",
```

Comment:
IIRC, @sethladd created a bunch of /+ shortcuts. For example, near me I see an old poster for lexical scope. At the bottom of it is the URL dartlang.org/+lexicalscope.